packages/@aws-cdk/aws-rds/lib/cluster-engine.ts

```diff
@@ -510,6 +548,13 @@ export interface AuroraPostgresEngineFeatures {
    * @default false
    */
   readonly s3Export?: boolean;
+
+  /**
+   * Whether this version of the Aurora Postgres cluster engine supports the lambda invocation feature.
+   *
```

Comment:
My comment here is basically the same as above, except, this could just be an array and the user inputs which features to include instead of true/false for each.

Reply:
Makes sense. Having all the `{ s3Import: true, s3Export: true, lambda: true }` lines below is very verbose.